flujo/infra/sandbox/null_sandbox.py

```
@@ -0,0 +1,22 @@
+
+from typing import Final
+
+from ...domain.sandbox import SandboxExecution, SandboxProtocol, SandboxResult
+
+
+class NullSandbox(SandboxProtocol):
+    """No-op sandbox used as a safe default when sandboxing is not configured."""
+
+    _SANDBOX_DISABLED_MSG: Final[str] = "Sandbox execution disabled (NullSandbox)."
+
+    async def exec_code(self, request: SandboxExecution) -> SandboxResult:
```

Ruff (0.14.7)

Unused method argument: `request`

(ARG002)